conda_smithy/templates/azure-pipelines-linux.yml.tmpl

```diff
@@ -11,7 +11,26 @@ jobs:
     fetchDepth: {{ clone_depth }}
 {%- endif %}
   - script: |
-         rm -rf /opt/ghc
+         sudo mkdir -p /opt/empty_dir
+         for d in /opt/hostedtoolcache \
+                  /usr/lib/jvm \
+                  /usr/local/.ghcup \
+                  /usr/local/android \
+                  /usr/local/powershell \
+                  /usr/share/dotnet \
+                  /usr/share/swift \
+                  ; do
+           sudo rsync -a --delete /opt/empty_dir/ $d
```

Comment:
```suggestion
           sudo rsync -a --delete /opt/empty_dir/`basename $d` $d || true
```

Reply:
I only create `/opt/emtpy_dir` and then use that for everything. Should I create a separate empty directory for each one?

Reply:
It might be ok since we have the trailing `/`

Maybe we could try adding these options `-v --stats --progress` to `rsync` and see if it is doing what we want there

Reply:
I linked a [log file](https://dev.azure.com/phenix-release/f0ee1b2f-77b3-4fa6-a2c5-97101b71b939/_apis/build/builds/6068/logs/55) with all the output, but I added just `--stats` to the command here so that a summary of the deleted files is shown for each directory.

Reply:
Ok this seems reasonable. Thanks for checking! 🙏